addons/l10n_gh/data/account.tax.group.csv

```diff
@@ -0,0 +1,4 @@
+id,name
+tax_group_vat_gh, Value Added Tax
+tax_group_wht_gh, Withholding Tax
```

Comment:
No need to group by different rates of withholding?

Reply:
Will confirm and see if grouping can be used